tests/code-splitting/subtests/between-new-worker-type/webpack/index.md

```diff
@@ -1,16 +1,9 @@
 ---
-result: fail
+result: partial
 ---
 
-Bundling Web Workers is possible in Webpack using one of the many add-ons like [worker-loader] or [worker-plugin], however there is no documented solution for accessing the underlying behavior: compiling a new entry starting from a given module and returning its bundle URL. There have been [some](https://github.com/webpack-contrib/worker-loader/pull/159) [attempts](https://github.com/webpack-contrib/worker-loader/pull/227) to support this in worker-loader, but at present the only available solution is [worker-plugin/loader]:
+It's possible to extend webpack's integrated Web Worker support to new or custom Worker types, as long as `importScripts()` is supported.
 
-```js
-import workerUrl from 'worker-plugin/loader!./my-worker';
-new Worker(workerUrl);
-```
+This makes it possible to bundle code for use in new worker-like environments like Worklets. Code splitting between the "host" and "child" sets of bundles is also supported. Currently, bundles are loaded using `importScripts()`, which means code splitting cannot be used with [Module Workers] or Worklets.
```

Comment:
This seems like it's saying one thing then the other. "You can use this for worklets", "You cannot use this for worklets".

Maybe just say that non-module workers are supported, but module workers and worklets are not. I'm kinda split on whether this counts as a 'partial', because new worker types seem to be module workers. WDYT @developit?

Reply:
Ah yeah that's my bad - I think I added the bit about importScripts usage without realizing it read as a contradiction. We could just drop the "like Worklets" line?

```suggestion
This makes it possible to bundle code for use in new worker-like environments like Service Worker. Code splitting between the "host" and "child" sets of bundles is also supported. Currently, bundles are loaded using `importScripts()`, which means code splitting cannot be used with [Module Workers] or Worklets.
```

Reply:
+1!